docs/V4-Migration-Guide.md

```diff
@@ -25,6 +25,16 @@ If you are upgrading from a version before 3.0.0, please see the [previous migra
 
 ### Breaking Changes:
 
+#### [Goodbye Projector Mixin]()
+
+As part of the new vdom, we've removed the `ProjectorMixin`. The `ProjectorMixin` has been one of the more confusing aspects within dojo, especially due to the need to create an instance. This is fundamentally different to how widgets were used everywhere else in the framework.
```

Comment:
This is fundamentally different to how widgets were used everywhere else in the framework. -> The previous use of `ProjectorMixin` was fundamentally different to how widgets were used everywhere else in the framework. The new approach for ___________ is now more straightforward and consistent.

(or if you feel the next paragraph is fine, ignore this suggestion)